### PR TITLE
22059 logging change

### LINKED ISSUE #22059: Porting a logging change into new CXF
Update log level to report the user_defined application error only when log level is or lower than FINE

### PR DESCRIPTION
**Pull request by BerksanAtes**
Fixes #22059

### CODE REVIEW

**Comment by BerksanAtes** [comment]
#build

File: dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/phase/PhaseInterceptorChain.java [hunk 1]
```diff
@@ -0,0 +1,841 @@
...
+                    description.append('#').append(opInfo.getName());
+                }
+                description.append("\' ");
+            }
+        }
+        return description.toString();
+    }
+
+    private void doDefaultLogging(Message message, Exception ex, String description) {
+        FaultMode mode = (FaultMode) message.get(FaultMode.class);
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/phase/PhaseInterceptorChain.java:398]: Can you mark this code change with the `//Liberty Change Start`

**Review by neuwerk — COMMENTED** [review]
Simple change, but want to make sure we track what ever done.

File: dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/phase/PhaseInterceptorChain.java (hunk 1 above)
Comment (BerksanAtes) [reply dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/phase/PhaseInterceptorChain.java:398]: Done

**Review by neuwerk — APPROVED** [review]

**Comment by BerksanAtes** [comment]
#libby-bot

**Pull request merged**